src/ray/common/ray_config_def.h

```diff
@@ -143,7 +143,7 @@ RAY_CONFIG(int64_t, worker_cap_max_backoff_delay_ms, 1000 * 10)
 /// The fraction of resource utilization on a node after which the scheduler starts
 /// to prefer spreading tasks to other nodes. This balances between locality and
 /// even balancing of load. Low values (min 0.0) encourage more load spreading.
-RAY_CONFIG(float, scheduler_spread_threshold, 0.5);
+RAY_CONFIG(float, scheduler_spread_threshold, 0);
```

Comment:
I thought we decided this was a bit dangerous to do right before 2.0 branch cut?

Reply:
This may have a bad effect on scaling down the cluster? Do you have the same concern? @wuisawesome

Reply:
yeah spreading out primary copies of objects in the short term could be bad for multiple reasons (also data locality).

Reply:
I think the conclusion offline was that we don't really another option to fix this issue. Please comment if you have one.

The downscaling issue is real, but I also don't think that this is a good reason to block this change. AFAIK, the load-balancing issue is the more concrete use case at the moment, and it is possible to fix the downscaling issue in other ways down the line. (Also, this change is arguably better for data movement since it will spread any subsequent requests based on data locality)